.github/workflows/coverage.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       run: |
         export OMPI_MCA_rmaps_base_oversubscribe=yes
         mpirun -n 2 -- coverage run -m pytest tests/integration_tests/test_chainermn.py
+        mpirun -n 2 -- pytest tests/integration_tests/test_pytorch_distributed.py
```

Comment:
Coverage seems recovered, but do we need to align it with `test_chainermn.py`?

```suggestion
        mpirun -n 2 -- coverage run -m pytest tests/integration_tests/test_pytorch_distributed.py
```

Reply:
Yes, you are right. Let me fix as you suggested.